app/backend/src/clj/teet/file/file_queries.clj

```diff
@@ -114,3 +112,28 @@
       ;; If metadata can't be parsed, return empty map, frontend will
       ;; know that filename is not valid
       {})))
+
+(defn valid-filename?
```

Comment:
valid filename for what? if this is only used for the exported zip, the name could reflect that.
It could also be more strict, so that the filename part must be valid uuid.
should also be private

Reply:
Added it as a precaution since the endpoint is open and the filename goes straight to the content-disposition, so it can't be messed around with. Maybe the s3-key should have some checking as well?

Reply:
Could also check for metacharacters (newline at least) that could be used to do header injection like this: https://owasp.org/www-community/attacks/HTTP_Response_Splitting

Reply:
The valid-chars-in-description? should not allow any extra characters in the filename